set-GET-beta/README.md

```diff
@@ -0,0 +1,25 @@
+# Switch GET tool registration to point atthe Beta version for Canvas Test and Beta - DO NOT USE ON PRODUCTION
```

Comment:
typo

Reply:
i corrected this and the other spelling mistakes in the README.md. Ta.